packages/web/components/cards/pool-card.tsx

```diff
@@ -0,0 +1,197 @@
+              height={54}
+            />
+          ) : (
+            <div
+              className={classNames(
+                "w-[54px] h-[54px] bg-blue rounded-full flex items-center justify-center",
+                poolCardIconBackgroundColorsToTailwindBgImage[
+                  PoolCardIconBackgroundColors[
+                    deterministicInteger % PoolCardIconBackgroundColors.length
+                  ]
+                ]
+              )}
+            >
+              <Image
+                src="/icons/OSMO.svg"
+                alt="no token icon"
+                width={40}
+                height={40}
+              />
+            </div>
+          )}
+        </div>
+        <div
+          className={
+            "ml-10 mr-6 w-[4.125rem] h-[4.125rem] rounded-full border-[1px] border-enabledGold shrink-0 flex items-center justify-center"
+          }
+        >
+          {pool.poolAssets.length >= 3 ? (
+            <div className="body1 text-white-mid ml-2.5">{`+${
+              pool.poolAssets.length - 1
+            }`}</div>
+          ) : pool.poolAssets[1].amount.currency.coinImageUrl ? (
+            <Image
+              src={pool.poolAssets[1].amount.currency.coinImageUrl}
+              alt={pool.poolAssets[1].amount.currency.coinDenom}
+              width={52}
+              height={52}
+            />
+          ) : (
+            <div
+              className={classNames(
+                "w-[54px] h-[54px] bg-blue rounded-full flex items-center justify-center",
+                poolCardIconBackgroundColorsToTailwindBgImage[
+                  PoolCardIconBackgroundColors[
+                    deterministicInteger % PoolCardIconBackgroundColors.length
+                  ]
+                ]
+              )}
+            >
+              <Image
+                src="/icons/OSMO.svg"
+                alt="no token icon"
+                width={40}
+                height={40}
+              />
+            </div>
+          )}
+        </div>
+        <div className="flex flex-col">
+          <h5>
+            {pool.poolAssets.length >= 3
+              ? `${pool.poolAssets.length} Token Pool`
+              : pool.poolAssets
+                  .map((asset) => asset.amount.currency.coinDenom)
+                  .join(" / ")}
+          </h5>
+          <div className="subtitle2 text-white-mid">{`Pool #${pool.id}`}</div>
+        </div>
+      </div>
+      <div className="mt-5 mb-3 w-full bg-secondary-200 h-[1px]" />
+      <div className="flex flex-wrap gap-x-8">
+        <div className="flex flex-col">
+          <div className="subtitle2 text-white-disabled">APR</div>
+          {queryOsmosis.queryIncentivizedPools.isAprFetching ? (
+            <div className="relative overflow-hidden rounded-sm w-[3.75rem] h-4 bg-white-faint mt-[0.4375rem]">
+              <div className="absolute left-0 -translate-x-[calc(-150%)] h-full w-1/2 bg-loading-bar animate-loading" />
+            </div>
```

Comment:
We may want to use this loading view in other places. Consider making a component that wraps contains this markup and accpets a boolean prop for whether it's fetching or not. If it's not fetching, it displays it's children. We called this kind of thing `<LoadingPlaceholder  isLoading={false}><children /></LoadingPlaceholder>` at my last job. But since this may only be used for loading individual metrics/value consider calling it `LoadMetricPlaceholder` or similar.